week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,69 @@
+
+
+
 // Write your code here
+//somehow get images in to that div
+//I can load first image initially with 
+//browser start and next button will load remaining images eventually as button pressed ?
+
+
+let imagesArray = ['image1.jpg', 'image2.jpg', 'image3.jpg', 'image4.jpg', 'image5.jpg', 'image6.jpg', 'image7.jpg'];
+//this is the div where I will display the images
+let sliderImage = document.querySelector('.sliderImage')
+let count = 0;
+
+function previousImage() {
+    
+    displayCounter.textContent = `This is an  image number ${count}`;
+    if (count <= 0) {
+    //if count passed the full array set the count equal to the length of the array.
```

Comment:
I love the fact that you've written comments on here in order to clarify the steps you've taken in your code 😄

Reply:
Thank you mitch for all your comments..